company_website/static/estimate_project/scripts/conversion_tracking.js

```diff
@@ -0,0 +1,16 @@
+function gtag_report_conversion(url) {
+    var callback = function () {
+        if (typeof(url) != 'undefined') {
+            window.location = url;
+        }
+    };
+    gtag(
+        'event',
+        'conversion',
+        {
+            'send_to': 'AW-607460288/pqT6CMvlzdkBEMC31KEC',
```

Comment:
I'm wondering if this should be a part of django settings because it is somehow production parameters

Reply:
So far the cleanest way in my opinion I've found so far would be to declare a variable in the head template, after passing the constant to the context:
```
%script
    window.conversion_tracking_target_address = {{ conversion_tracking_target_address }}
```
Then I can access that from the .js files. Would this solution be acceptable?